src/Layer.js

```diff
@@ -332,6 +336,10 @@ export default class Layer {
             : [];
     }
 
+    isAnimated () {
+        return this._isAnimated;
```

Comment:
I am not sure of this, so this is more a question: I think isAnimated method should be a getter:

```js
get isAnimated () {
    return this._viz.isAnimated()
}
```

We will not need then `_isAnimated` property.

Reply:
Yes. I see that I added the flag because of the first implementation (in order to subscribe only to animated layers) but with the final implementation, the evaluation of the animated layer is in the updated event callback so the flag is not needed anymore.